Refresh.GameServer/Types/Assets/GameAsset.cs

```diff
@@ -23,4 +32,341 @@ public partial class GameAsset : IRealmObject
     public IList<string> Dependencies { get; } = null!;
 
     [Ignored] public AssetSafetyLevel SafetyLevel => AssetSafetyLevelExtensions.FromAssetType(this.AssetType);
+
+    public string? AsMainlineIconHash { get; set; }
+    public string? AsMipIconHash { get; set; }
+    
+    //NOTE: there's no "as MIP photo hash" because theres no way to browse photos on LBP PSP.
+    public string? AsMainlinePhotoHash { get; set; }
+
+    /// <summary>
+    /// Transforms an input image into a photo-friendly format
+    /// </summary>
+    /// <param name="image">The input photo</param>
+    /// <returns>The transformed image</returns>
+    private Image<Rgba32>? PhotoTransformation(Image image)
+    {
+        //No transformation needs to be done for photos.
+        return null;
```

Comment:
This function definitely shouldn't be here. If we ever have the need to do transformations on photos then we should just write the function then.